### PR TITLE
Update documentation about TESTDEBUG usage

### PR DESCRIPTION
**Pull request by aaannz**
- documentation for https://github.com/os-autoinst/os-autoinst/pull/374

### CODE REVIEW

File: docs/WritingTests.asciidoc [hunk 1]
```diff
@@ -195,31 +195,67 @@ settings:
...
+Using snapshots to speed up development of tests
 ------------------------------------------------
 [id="snapshots"]
 
 Sometimes it's annoying to run the full installation to adjust some
-test. It would be nice to have the VM jump to a certain point. There
-is an experimental hidden feature that allows to start from a
-snapshot that might help in that situation:
+test. It would be nice to have the VM jump to a certain point. +QEMU
+backend+ provides feature that allows job to start from a snapshot that might
```
Comment (okurz) [inline docs/WritingTests.asciidoc]: "provides *a* feature that allows *a* job to start"

File: docs/WritingTests.asciidoc [hunk 2]
```diff
@@ -195,31 +195,67 @@ settings:
...
+help in this situation.
 
+Depending on use case, there are two options to help:
```
Comment (okurz) [inline docs/WritingTests.asciidoc]: "on **the** use case"

File: docs/WritingTests.asciidoc [hunk 3]
```diff
@@ -195,31 +195,67 @@ settings:
...
+are in the image. Snapshots are named `"test module category"-"test module name"` (e.g. `installation-start_install`)
+----
+
+.Steps to enable storing only last successful snapshot (latter mode):
+----
+1. run the worker with --no-cleanup parameter. This will preserve the hard
+disks after test runs.
+
+2. set +TESTDEBUG=1+ on a job. This will make openQA save a
+snapshot for after successful test module run. Snapshots are overwritten.
```
Comment (okurz) [inline docs/WritingTests.asciidoc]: /for/d

**Comment by okurz** [comment]
minor changes, then LGTM

**Comment by aaannz** [comment]
fixed, ready for round 2

**Comment by okurz** [comment]
LGTM

**Pull request merged**